README.md

```diff
@@ -37,7 +37,7 @@ This crate supports the following metrics provided by [Prometheus] for
 For each platform, it is equivalent to what the official Prometheus client for
 Go ([client_golang]) provides.
 
-> [!NOTE]
+> NOTE
```

Comment:
I've reverted this.  Be aware however that this is not valid cargo doc syntax and it will emit an warning for this since the readme is included as module docs.

Reply:
Yeah, but I think it’s not a big deal.